libenkf/include/ert/enkf/site_config.h

```diff
@@ -101,6 +103,11 @@ typedef struct site_config_struct site_config_type;
   mode_t                   site_config_get_umask( const site_config_type * site_config );
 
   site_config_type       * site_config_alloc_empty();
+  site_config_type       * site_config_alloc_default();
+  site_config_type       * site_config_alloc_model_config(const char *);
```

Comment:
This allocates a `site_config` instance - i.e. it should be named something like `site_config_alloc( )`; the current name seems to indicate that this function will create a `model_config` structure from a `site_config` - that is not the case.